app/views/pages/admin.html.slim

```diff
@@ -0,0 +1,22 @@
+main.site-admins
+  section.section
+    header.has-text-centered
+      h1.is-size-1 Admin Members
+      p You can contact any of the following administrators
+
+    main.container
+      .columns.is-multiline
+        - @admin_members.each do |member|
+          .column.is-12-mobile.is-6-tablet.is-4-desktop
+            .card.has-text-centered
+              .card-image = image_tag member.data["profile"]["image_192"]
+
+              .card-content
+                h3.title
+                  = member.data["profile"]["real_name"]
+
+              .card-footer
+                .card-footer-item
+                  = link_to "mailto:#{member.data['profile']['email']}", class: "button is-fullwidth is-medium" do
```

Comment:
I'm not sure we want to put our emails online in this way.  I get enough spam as it is.  I think it would be better if we could add a web form that would send email to us after appropriate captcha, etc., but I realize that's a lot more work too.

For now, can we replace the mailto with the admin's slack username?  It won't help external users, but it will help members of the slack team.

Thoughts?

Reply:
I'd rather not have my email be made publicly accessible as well.